source/products/hansemerkur/tchibo/cat-surgery/webapp.rst

```diff
@@ -0,0 +1,22 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://js.kasko.io"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        key: "KEY",
+        touchpoint: "tp_d0f608514295b1a6f2ae07e769b34",
+        mode: "fullscreen",
+        language: "de",
+        config: {
+          adnr_number: "{adnr_number}"
```

Comment:
Hmm does tchibo also have adnr_number? 🤔

Reply:
It does not? 🙈

Reply:
Dont know 🤷‍♀️

Reply:
Asked PM, they probably use it. so all good

Reply:
Cool, thanks!